CONTRIBUTING.md

```diff
@@ -22,6 +22,25 @@ When filing an issue, please use this template:
     # Additional Information:
 
 
+Rules and guidelines for developers
+-----------------------------------
+
+Please create one branch (named accordingly) for each different feature/exchange you are working on and submit a separate PR for each one of them, like this it's easier to review and approve the PR.
+
+If you just forked the project and you want to add your changes to the _dev_ branch, don't do that: create a new branch based on _dev_, instead.
+
+E.g. if you are implementing a new exchange called BitFoo, create a branch called _bitfoo_.
+
+If you are fixing some bugs in the _CryptoBar_ exchange, create a branch called _cryptobar-fix_ (or just _cryptobar_).
+
+When the PR is merged, you can delete these branches. Like this, it's also easier for you to [sync your forked dev branch with the upstream one](https://help.github.com/articles/syncing-a-fork/).
+
+- [Standardized Methods](README.md#user-content-standardized-methods) signatures should all have the required parameters, especially [*args and **kwargs](https://github.com/Crypto-toolbox/bitex/issues/183) (even if they are not used. Your IDE may complain: in that case you can add a comment to ignore the warning).
+- Method signatures should have positional parameters first, then *args (if required), then optional parameters, e.g. see bitstamp interface: `def wallet(self, *args, pair=None, **kwargs)`
+- Configure your IDE to show visual guides at 100 columns and write code that doesn't exceed 100 characters per line.
+- For string formatting, for **new files** please use the notation `'%s-%s'.str(foo, bar)` (in _old/existing_ files we may have used the notation `'%s' % (foo, bar)`: in that case, either use the same notation or update all the formattings you see, while working on it).

```

Comment:
Always use `str.format()` and replace any old-style formatting with this.

Reply:
Done